config/examples/redis.yml

```diff
@@ -5,6 +5,14 @@ base: &default
   sentinels: "<%= ENV['REDIS_SENTINEL_HOSTS'] %>"
   name: <%= ENV['REDIS_SENTINEL_NAME'] %>
   role: <%= ENV['REDIS_SENTINEL_ROLE'] %>
+  # == ACL Params ==
+  username: <%= ENV['REDIS_ACL_USERNAME'] %>
```

Comment:
Hm, I would call the env vars just `REDIS_USERNAME` and `REDIS_PASSWORD` (with `BACKEND_` prefix accordingly), I don't think we need to mention in the env var itself that it's about ACL (it's good to have it in the comment though)